presto-docs/src/main/sphinx/sql/show-grants.rst

```diff
@@ -0,0 +1,42 @@
+===========
+SHOW GRANTS
+===========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW GRANTS ON ( [ TABLE ] table_name | ALL )
+
+Description
+-----------
+
+List the grants for the current user on the specified table in the current catalog.
+
+Specifying ``ALL`` lists the grants on all the tables in the current catalog.
```

Comment:
What schema is used if none is given?
Does it list all tables in all schemas in given catalog?